addon/components/frost-action-bar.js

```diff
@@ -22,7 +22,7 @@ export default Component.extend({
 
   propTypes: {
     // options
-    controls: PropTypes.arrayOf(PropTypes.EmberComponent).isRequired,
```

Comment:
If you don't want to update the linting now add `// eslint-disable-line` instead of changing it to `any`